pkg/alertmanager/alertmanager.go

```diff
@@ -42,6 +44,23 @@ import (
 
 const notificationLogMaintenancePeriod = 15 * time.Minute
 
+// NilPeer and NilChannel implements the Alertmanager clustering interface used by the API to expose cluster information.
```

Comment:
Nit: I would move this type to the bottom of the file, to keep most important types on the top.